src/include/msgr.h

```diff
@@ -9,14 +9,8 @@
  * Data types for message passing layer used by Ceph.
  */
 
-#define CEPH_MON_PORT    6789  /* default monitor port */
-
-/*
- * client-side processes will try to bind to ports in this
- * range, simply for the benefit of tools like nmap or wireshark
- * that would like to identify the protocol.
- */
-#define CEPH_PORT_FIRST  6789
+#define CEPH_MON_PORT         3300  /* (ce4h) IANA assigned monitor port */
```

Comment:
@runsisi  ce4h means ce4 in Hex format, which is equivalent to 3300 in Dec format.